tools/StaticAnalysis/Exceptions/ValidateHelpIssues.csv

```diff
@@ -0,0 +1,142 @@
+Target,Description
```

Comment:
May I know why you added this file?

Reply:
This was a file that got generated when I was trying to build the help files via one of our tools.  I removed it.